tracer/build/_build/Build.Steps.cs

```diff
@@ -2418,6 +2418,11 @@ string NormalizedPath(AbsolutePath ap)
                new(@".*Exception occurred when calling the CallTarget integration continuation. Datadog.Trace.ClrProfiler.CallTarget.CallTargetInvokerException: Throwing a call target invoker exception.*"),
                // error thrown to check error handling in RC tests
                new(@".*haha, you weren't expect this!*", RegexOptions.Compiled),
+               // known errors in waf config
+               new(@".*rc::asm_dd::diagnostic Error: missing key.*", RegexOptions.Compiled),
```

Comment:
It would be nice if we could scope these to specific test runs/files, as if we get these errors in _other_ tests that in it indicates a problem 🤔